src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -97,6 +114,20 @@ const SendRequest = () => {
     }
   };
 
+  const draftPackageHandler = () => {
+    const newData = {
+      requestId,
+      newStage: "PackageApproval",
+      newSubStage: "SelectionPackageOSFApproval",
+      eventTitle:
+        "Forward Stage Change: Draft Package (HRL) to Package Approval",
```

Comment:
Now that you've moved where it calculates what the next phase is, this is simply using that next phase and appending on the extra couple fields.  That way the phase and event text all match your constant STAGES.

```suggestion
    const newData = {
      requestId,
      newStage: nextStage,
      newSubStage: nextSubStage,
      eventTitle: eventText,
```